src/dailies.ts

```diff
@@ -274,6 +275,8 @@ function dailyBuffs(): void {
     ensureEffect($effect`Down the Rabbit Hole`);
     cliExecute("hatter 22");
   }
+
+  BeachComb.tryHead($effect`Do I Know You From Somewhere?`);
```

Comment:
Will this get the effect if the free combs are used and it would cost an adventure?

Reply:
yes it would. We probably want to do that no matter what though, right?

Reply:
If that's the case it should be done first so we don't burn a turn of all the other daily buffs acquired before it.